### ISSUE TITLE
Problems releasing after upgrade to v3

### ISSUE THREAD

**Issue by JyrkiHei** [Azure/webapps-deploy#374]
Hello! I have a flow that updates App Service web app running on premium v3 plan. I updated webapps-deploy action to V3 and the flow stopped working. After dropping the version back to v2 the flow started working again. The flow is using OIDC-authentication.

Run azure/webapps-deploy@v3
  with:
    app-name: wa-euw-qwerty-backend-dev
    package: ./build.zip
    type: zip
    slot-name: production
  env:
    AZURE_HTTP_USER_AGENT: 
    AZUREPS_HOST_ENVIRONMENT: 

Updating submodules.
Preparing deployment for commit id '6084cadb-6'.
PreDeployment: context.CleanOutputPath False
PreDeployment: context.OutputPath /home/site/wwwroot
Skipping build. Project type: Run-From-Zip
Skipping post build. Project type: Run-From-Zip
Requesting site restart
Requesting site restart. Attempt #1
Requesting site restart. Attempt #2
Requesting site restart. Attempt #3
Requesting site restart. Attempt #4
Requesting site restart. Attempt #5
Failed to request a restart. Number of attempts: 5. Exception: System.Net.Http.HttpRequestException: The SSL connection could not be established, see inner exception.\n ---> System.IO.IOException:  Received an unexpected EOF or 0 bytes from the transport stream.\n   at System.Net.Security.SslStream.<FillHandshakeBufferAsync>g__InternalFillHandshakeBufferAsync|189_0[TIOAdapter](TIOAdapter , ValueTask`1 , Int32 )\n   at System.Net.Security.SslStream.ReceiveBlobAsync[TIOAdapter](TIOAdapter )\n   at System.Net.Security.SslStream.ForceAuthenticationAsync[TIOAdapter](TIOAdapter , Boolean , Byte[] , Boolean )\n   at System.Net.Http.ConnectHelper.EstablishSslConnectionAsync(SslClientAuthenticationOptions , HttpRequestMessage , Boolean , Stream , CancellationToken )\n   --- End of inner exception stack trace ---\n   at System.Net.Http.ConnectHelper.EstablishSslConnectionAsync(SslClientAuthenticationOptions , HttpRequestMessage , Boolean , Stream , CancellationToken )\n   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage , Boolean , CancellationToken )\n   at System.Net.Http.HttpConnectionPool.CreateHttp[11](https://github.com/Company-name/qwerty-backend/actions/runs/6768066904/job/18391824203#step:4:12)ConnectionAsync(HttpRequestMessage , Boolean , CancellationToken )\n   at System.Net.Http.HttpConnectionPool.AddHttp11ConnectionAsync(HttpRequestMessage )\n   at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken )\n   at System.Net.Http.HttpConnectionPool.GetHttp11ConnectionAsync(HttpRequestMessage , Boolean , CancellationToken )\n   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage , Boolean , Boolean , CancellationToken )\n   at System.Net.Http.DiagnosticsHandler.SendAsyncCore(HttpRequestMessage , Boolean , CancellationToken )\n   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage , Boolean , CancellationToken )\n   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage , HttpCompletionOption , CancellationTokenSource , Boolean , CancellationTokenSource , CancellationToken )\n   at Kudu.Core.Helpers.PostDeploymentHelper.PostAsync(String path, String requestId, String content) in /tmp/KuduLite/Kudu.Core/Helpers/PostDeploymentHelper.cs:line 6[29](https://github.com/Company-name/qwerty-backend/actions/runs/6768066904/job/18391824203#step:4:30)\n   at Kudu.Core.Helpers.PostDeploymentHelper.<>c__DisplayClass38_0.<<RestartMainSiteAsync>b__0>d.MoveNext() in /tmp/KuduLite/Kudu.Core/Helpers/PostDeploymentHelper.cs:line 350\n--- End of stack trace from previous location ---\n   at Kudu.Core.Infrastructure.OperationManager.<>c__DisplayClass4_0.<<AttemptAsync>b__0>d.MoveNext() in /tmp/KuduLite/Kudu.Core/Infrastructure/OperationManager.cs:line 56\n--- End of stack trace from previous location ---\n   at Kudu.Core.Infrastructure.OperationManager.AttemptAsync[TVal](Func`1 action, Int[32](https://github.com/Company-name/qwerty-backend/actions/runs/6768066904/job/18391824203#step:4:33) retries, Int32 delayBeforeRetry) in /tmp/KuduLite/Kudu.Core/Infrastructure/OperationManager.cs:line 67\n   at Kudu.Core.Helpers.PostDeploymentHelper.RestartMainSiteAsync(String requestId, TraceListener tracer) in /tmp/KuduLite/Kudu.Core/Helpers/PostDeploymentHelper.cs:line [34](https://github.com/Company-name/qwerty-backend/actions/runs/6768066904/job/18391824203#step:4:35)4
Deployment Failed. deployer = OneDeploy deploymentPath = OneDeploy
Error: Failed to deploy web package to App Service.
Error: Deployment Failed, Package deployment using ZIP Deploy failed. Refer logs for more details.
App Service Application URL: https://wa-euw-qwerty-backend-dev.azurewebsites.net/

**Comment by sgollapudi77** [comment]
Hey @JyrkiHei can you confirm if this is your app name : `wa-euw-qwerty-backend-dev`, because we couldn't find any logs in our system on this deployment.

**Comment by JyrkiHei** [comment]
@sgollapudi77  No, I changed the name to "hide" the actual environments. I can send you the real name with email or something?

**Comment by dannysongg** [comment]
Hi @JyrkiHei 
Is this issue still occurring? If so, please send me an email (dannysong@microsoft.com) with the name of the webapp you are trying to deploy to.
